src/features/offer/pages/Offer.tsx

```diff
@@ -35,12 +41,34 @@ const OfferComponent: FunctionComponent = () => {
   const headerScroll = useRef(new Animated.Value(0)).current
   const hasSeenAllPage = useRef<boolean>(false)
 
+  const appState = useRef(AppState.currentState)
+  const timeInBackground = useRef(0)
```

Comment:
```suggestion
  const timeInBackground = useRef(0).current
```
pour que ce soit plus lisible ensuite

Reply:
pareil pour startTimeBackground